.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,39 @@
-Issue:
+<!-- Thank you for contributing to Storybook! If your PR is related to an issue, provide the number(s) here, e.g. #1000, #1001 -->
+Issue: #
```

Comment:
I feel like @shilman always changes this to just be the issue number without the "Issue:", to make it work with the release script. If that is the case, then I guess we should just remove it from here.